src/device-registry/README.md

```diff
@@ -38,10 +39,21 @@ Build the image within the microservice's directory
 docker build -t {IMAGE_NAME} .
 ```
 Then run the container based on the newly created image
-
 ```bash
 docker run -d -n {CONTAINER_NAME} -p host-port:container-port {IMAGE_NAME}
 ```
+To build all images
+```bash
```

Comment:
This looks interesting, thanks! :) 
What is the motivation for these bash scripts?
Under what scenario would one want to build all images at once?

Reply:
When peforming deployment operations